index.html

```diff
@@ -4,28 +4,48 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>myProject</title>
+    <title>MyProject</title>
     <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="popUp.css" />
+    <script defer src="menu.js"></script>
+    <script defer src="popUp.js"></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
   </head>
 
   <body>
-    <header class="header">
-      <nav class="header-container">
+    <header class="navibar">
+      <nav class="nav-container">
         <a id="logo" class="logo" href="#">My Logo</a>
-        <ul class="header-list">
-          <li class="header-item"><a href="#Portfolio">Portofolio</a></li>
-          <li class="header-item"><a href="#About">About</a></li>
-          <li class="header-item"><a href="#Contact">Contact</a></li>
+        <ul class="nav-list">
+          <li class="navi-item"><a href="#Portfolio">Portofolio</a></li>
+          <li class="navi-item"><a href="#About">About</a></li>
+          <li class="navi-item"><a href="#Contact">Contact</a></li>
         </ul>
-        <button type="button" id="menu-btn" class="menu">
+        <button type="button" id="menu-btn">
           <img class="mobile-menu" src="images/Union.png" alt=" navigation" />
```

Comment:
- [ ] Kindly make sure you mobile menu design sticks as much as possible to the one of your chosen Figma template design. e.g.: `fonts(size, weight,)`, `line height` on options text, spacing `between` and `around` all options text, spacing around all elements,... Also try to improve the **background blur** `effect`to match the expected design. **Make sure you check all details from the Figma template and stick as much as possible the expected design**

| Figma Design | Your Design |
| ------------------ | ----------------- |
| ![Screenshot from 2023-01-04 13-57-26](https://user-images.githubusercontent.com/83042742/210551503-936efc1c-6690-4184-9f67-8c873aade600.png) | ![Screenshot from 2023-01-04 14-06-27](https://user-images.githubusercontent.com/83042742/210551739-c9137df8-3717-47c1-ada5-ba5cf3ecddf2.png) |